target/linux/rockchip/armv8/config-5.4

```diff
@@ -367,6 +367,7 @@ CONFIG_RCU_CPU_STALL_TIMEOUT=21
 # CONFIG_RCU_EXPERT is not set
 CONFIG_RCU_TRACE=y
 # CONFIG_READ_ONLY_THP_FOR_FS is not set
+CONFIG_REALTEK_PHY=y
```

Comment:
This is already done here?
https://github.com/openwrt/openwrt/commit/4b4bff50706ea16de6146839973e56308dfb0af4

Reply:
You are right, I missed that and removed it from the commit